include/eve/module/core/saturated/core.hpp

```diff
@@ -10,10 +10,8 @@
 
 #include <eve/module/core/saturated/add.hpp>
 #include <eve/module/core/saturated/convert.hpp>
-#include <eve/module/core/saturated/dec.hpp>
```

Comment:
Only those are removed ? We modified a lot of functions here, I woudl have expected more removals.

Reply:
found 2 more